tensorflow_datasets/image/caltech10k_webfaces.py

```diff
@@ -0,0 +1,125 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Caltech 10K Web Faces.
+
+The dataset contains images of people collected from the web by typing common given names into  Google Image Search
+
+Pruning Training Sets for Learning of Object Categories
+Anelia Angelova, Yaser Abu-Mostafa, Pietro Perona
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+
+import tensorflow_datasets.public_api as tfds
+
+LANDMARK_HEADINGS = ("lefteye_x lefteye_y righteye_x righteye_y "
+                     "nose_x nose_y mouth_x mouth_y").split()
+IMAGE_DATA = ("http://www.vision.caltech.edu/Image_Datasets/Caltech_10K_WebFaces/Caltech_WebFaces.tar")
+LANDMARK_DATA = ("http://www.vision.caltech.edu/Image_Datasets/Caltech_10K_WebFaces/WebFaces_GroundThruth.txt")
+
+
+_DESCRIPTION = """\
+The dataset contains images of people collected from the web by typing common given names into 
+Google Image Search. The coordinates of the eyes, the nose and the center of the mouth for each 
+frontal face are provided in a ground truth file. This information can be used to align and crop 
+the human faces or as a ground truth for a face detection algorithm. The dataset has 10,524 human 
+faces of various resolutions and in different settings, e.g. portrait images, groups of people, etc. 
+Profile faces or very low resolution faces are not labeled.
+""" 
+class Caltech10K_WebFaces(tfds.core.GeneratorBasedBuilder):
+
+    VERSION = tfds.core.Version("0.1.0")
```

Comment:
Indentation should be 2 as in other files.

Reply:
Done